### ISSUE TITLE
Advanced view: content that opens in a column should open a new column // don't cover the menu column

### ISSUE THREAD

**Issue by brendanjones** [mastodon/mastodon#21671]
### Pitch

When opening any item from the menu column, it will open in that same column, covering the menu. Only if you pin the column does it move out into its own column and uncover the menu.

I propose that the menu column should never be covered. This would mean all content that opens in a column should always open a new column. 

The items that'd open in a new column:

1. Explore
2. Local timeline
3. Federated timeline
4. Home
5. Direct messages
6. Bookmarks
7. Favourites
8. Lists
9. About
10. Profiles directory
11. Privacy policy
12. Keyboard shortcuts
13. 'Trending now' hashtags
14. Anything selected from search (hashtags, user profiles, posts)

'Settings' would (still) not open in a column.

### Motivation

1. It'd make the 'pin' action more intuitive. If I pin a column that's on top of the menu, where will it go when I pin it? Having it already in its own column removes this question.
2. Temporarily hiding the menu isn't great UX.
3. You could move the pin/unpin action out of the settings and keep it at the top of the pinnable columns, so it's immediately obvious which columns are/aren't pinned, and that you *can* pin.

**Comment by eishiya** [comment]
By "menu", do you mean the Getting Started column? I think a better solution would be to make that column (and really, any column...) pinnable, so that users who want it can have it available at all times.

I find the current approach of having any number of static pinned columns and one dynamic column that has your currently viewed content quite convenient, spawning new columns would just require constant horizontal scrolling and closing the previous column when you want to move on.

**Comment by brendanjones** [comment]
Yes, I mean the Getting Started column. Not sure why it's even got that name tbh, I'd suggest removing that. For consistency with the simple view it should be replaced with the Mastodon logo.

Most columns already are pinnable, so I don't see how that's an issue?

As long as the new column opened to the left of the menu column then you wouldn't need to horizontal scroll. Scrolling would work exactly the same as it works now when you open something from search: your scroll position is moved to focus on that column. Right now it opens on top of the menu column, whereas in my proposal a new column is opened and the menu column is pushed to the right. If you don't have many columns open then your scroll position wouldn't even move. If you did have many columns open then you'd need to scroll anyway. 

The only downside I can see is that this might mess with web browser navigation. As in, opening a new column opens a new url, so you can close a column by navigating back. I suspect you could maintain this interaction, however.
